### ISSUE TITLE
 In ex1_multi, why is my house price prediction not the same for both methods?

### ISSUE THREAD

**Issue by HasakiWMC** [HasakiWMC/ex1#1]
Q8) In ex1_multi, why is my house price prediction not the same for both methods?

If the two predictions are very far apart, it probably means you did not normalize the features of the prediction when using theta from the gradient descent method. This means you should:

create a vector of the prediction features, such as "[1650 3]".
subtract 'mu', then divide element-wise by 'sigma'. mu and sigma were returned by featureNormalize().
add a bias unit
multiply by theta
If the two predictions are off by a few percent, then you need to modify the learning rate and the number of iterations in ex1_multi.m. If you handle the normalization correctly and pick good values for the learning rate and number of iterations, then you can get the same price for both GD and the NE method.

**Comment by HasakiWMC** [comment]
预测向量也要进行特征缩放！！！
